.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,6 +37,7 @@ This checklist is meant to remind the package maintainer(s) who will review this
 - [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
 - [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
 - [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
+- [ ] Do the commits need to be squashed or rebased? This is only important if the milestone is for bugfix because we need a clean merge commit for the backport (no "Squash and merge").
```

Comment:
are we *sure* this is true?  Can we test on a case where we don't care if it fails first?

Reply:
Actually with the bot, I *think* it can do "squash merge". But if we ever want to use astropy's old consistency check script, then pretty sure this is true, but if not, then maybe not...

So, do we want to script consistency check, or we YOLO and say "if you forget to backport, it won't be in bugfix release, too bad, it's on you"?

Reply:
@kecnry , I have removed this for now. We can revisit if squash & merge give us problem. Do you have any comment on the other updated list item?